ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -215,6 +216,17 @@ void KauthHandler_HandleKernelMessageResponse(uint64_t messageId, MessageType re
             }
             Mutex_Release(s_outstandingMessagesMutex);
         }
+        
+        // The follow are not valid responses to kernel messages
+        case MessageType_Invalid:
+        case MessageType_UtoK_StartVirtualizationInstance:
+        case MessageType_UtoK_StopVirtualizationInstance:
+        case MessageType_KtoU_EnumerateDirectory:
+        case MessageType_KtoU_HydrateFile:
+        case MessageType_KtoU_NotifyFileModified:
+        case MessageType_KtoU_NotifyFilePreDelete:
+        case MessageType_KtoU_NotifyDirectoryPreDelete:
+            break;
```

Comment:
Panic? Trace? Disconnect the provider?

Reply:
At a minimum we should trace.  

As a side-note, I made this change because after enabling "treat warnings as errors" (`"-Werror"`) we started getting compilation errors here (having unhandled enum values in a switch statement triggers a warning).